tests/unit/storage_v2_analytics_mode.cpp

```diff
@@ -0,0 +1,71 @@
+#include <gtest/gtest.h>
+#include <string_view>
+
+#include "storage/v2/isolation_level.hpp"
+#include "storage/v2/storage.hpp"
+#include "storage/v2/vertex_accessor.hpp"
+
+namespace {
+
+int64_t VerticesCount(memgraph::storage::Storage::Accessor &accessor) {
+  int64_t count{0};
+  for ([[maybe_unused]] const auto &vertex : accessor.Vertices(memgraph::storage::View::OLD)) {
+    ++count;
+  }
+  return count;
+}
+
+inline constexpr std::array storage_modes{
+    memgraph::storage::StorageMode::IN_MEMORY_ANALYTICAL,
+    memgraph::storage::StorageMode::IN_MEMORY_TRANSACTIONAL,
+};
+
+}  // namespace
+
+std::string_view StorageModeToString(memgraph::storage::StorageMode storage_mode) {
```

Comment:
Completely unimportant but as a side note what I learnt yesterday, maybe you will find it interesting. string_view can be constexpr while string cannot in c++17. This will change in c++20